src/System.Runtime/tests/System/Version.cs

```diff
@@ -192,7 +196,8 @@ private static IEnumerable<object[]> Parse_Valid_TestData()
         yield return new object[] { "1.2", new Version(1, 2) };
         yield return new object[] { "1.2.3", new Version(1, 2, 3) };
         yield return new object[] { "1.2.3.4", new Version(1, 2, 3, 4) };
-        yield return new object[] { "2  .3.    4.  \t\r\n15  ", new Version(2, 3, 4, 15) };
+        yield return new object[] { "   2  .3.    4.  \t\r\n15  ", new Version(2, 3, 4, 15) };
```

Comment:
Why did you add the spaces to the beginning?  Should this case be added as an additional input rather than replacing the existing one?